### ISSUE TITLE
The grey border on bundles is ugly

### ISSUE THREAD

**Issue by louthy** [leggett/simplify#687]
After being furious with Google for cancelling Inbox, I've been looking for an Inbox clone ever since.  I was so happy to finally find this project after many years, so thank you so much for the hard work.  It's worth money every day of the week!

I decided to try out the beta.  I assume the look-n-feel is still WIP, so I am a little tentative to post this.  But the two pixel grey border that shows when the bundle opens doesn't feel anywhere near as slick and elegant as the other features of Simplify.  

* The left/right borders probably aren't needed at all
* There's already a title segregator at the top, so probably no need for a top border
* There does need to be a bottom segregator, but it maybe doesn't need to be so dark

Selfishly (because I'm running Gmail with a background image), I'd much prefer some transparent space top n bottom to show the bundle 'disconnect' from the emails around it.  

I realise that doesn't work for the white-on-white or black-on-black schemes.  What I really liked about Inbox was the two-tone scheme, with a light grey background, with white foreground 'cards'.  It gave the various sections, bundles, days a feeling of separation that was really elegant and simple.

If there was a subtle difference in the border colours for those schemes (light grey bg for the white-on-white, slightly less black fg for the black-on-black) then those segregators would appear automatically if there was a space.

Or maybe this could be a configurable option of some sort.  But I'm not sure of the implications of that with Gmail's settings.

Anyway, again thank you for the work.  I'm a happy customer! 

![image](https://user-images.githubusercontent.com/1991827/172054889-e49f1ad9-d49c-47ca-a11d-9b3a6a8c7dd9.png)

---
Simplify v.3.0.20 - Configuration: aAX boldHighlight bundleInbox ca3 caI caL cfC cfO cfQ cfR cfU composeActions composeFab dateGroup fontList fontMsg hideEmoji hideLabelChips hideListCount hideMsgCount hideSelectRefresh hideTabIcons inInbox inList invertAddons invertCompose izBgDefault listWidthLg lowDensity mediumTheme minimizeChat msgLabelsRight msgWidthLg newUI reverseMsgs rightStar showAvatars simplify simplifyCanary supExtWarn - System: Mozilla/5.0 (Windows NT 10.0 Win64 x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.63 Safari/537.36 - Window: 2048 x 1104 (100pct zoom) - Language: en-GB

**Comment by leggett** [comment]
I agree and have trying to find something that looks better (esp when using a theme) on my list. I wanted to use a soft drop shadow but that proved hard/impossible based on how things are structured (there isn't a container around the bundled items). 

I was going to try adding a gap (such that the theme background shows through) instead of a border next.

**Comment by leggett** [comment]
Not perfect (it would take a lot to round the corners of the list above and below the bundle) but I think this is better. I'll push this in the next update (v3.0.21) and see what people think. (I've started working on bundle settings so it might be a week or so before this update goes out)

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/8763662/172076868-4e415753-9fcb-41da-8f18-1a40ef7776d6.png">

**Comment by louthy** [comment]
That's awesome! With or without rounded corners I think that's a massive improvement, and is definitely in the same league as the rest of the UX.
